etc/greenboot/green.d/00_grub_fallback.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+# We need to determine whether this boot was a fallback boot
+if grub2-editenv list | grep -q "fallback_active=true"; then
+  grub2-editenv - unset fallback_active
+  grub2-editenv - set boot_success=0 # this should already be 0
+  grub2-editenv - set boot_counter=2
```

Comment:
If we're doing a rollback, then we want to set `boot_success=1` so that the first entry (using 1-base index here :)) is always picked right? (Where the first entry is now the entry we're booted into when we're executing this).